src/System.Runtime.InteropServices.RuntimeInformation/pkg/win/System.Runtime.InteropServices.RuntimeInformation.pkgproj

```diff
@@ -9,8 +9,22 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\src\System.Runtime.InteropServices.RuntimeInformation.builds">
-      <AdditionalProperties>FilterToOSGroup=Windows_NT</AdditionalProperties>
+    <ProjectReference Include="..\..\src\System.Runtime.InteropServices.RuntimeInformation.csproj">
+      <OSGroup>Windows_NT</OSGroup>
+    </ProjectReference>
+    <ProjectReference Include="..\..\src\System.Runtime.InteropServices.RuntimeInformation.csproj">
+      <OSGroup>Windows_NT</OSGroup>
+      <TargetGroup>net45</TargetGroup>
+    </ProjectReference>
+    <ProjectReference Include="..\..\src\System.Runtime.InteropServices.RuntimeInformation.csproj">
+      <OSGroup>Windows_NT</OSGroup>
+      <TargetGroup>netcore50</TargetGroup>
+      <AdditionalProperties>PackageTargetPath=runtimes/win7/lib/netcore50</AdditionalProperties>
```

Comment:
I see.  We should clean up the use of AdditionalProperties & PackageTargetPath.  Instead of injecting the PackageTargetPath property we just need to have the project set the right PackageTargetRuntime and PackageTargetFramework for TargetGroup netcore50 and netcore50aot.